inst/base/modelheader.h

```diff
@@ -82,6 +82,8 @@ typedef double capture;
 #define AMT self.amt
 // Data set cmt
 #define CMT self.cmt
+// Bool flag indicating that the system is advancing to steady-state
+#define SS_ADVANCE _ss_flag_
```

Comment:
Or maybe `SS_ADVANCE` is intended for user to flag that ODE should keep evolving until `dxdt=0`?

Reply:
I'm not crazy about the names here.  Basically, I wanted to pass a flag to that RHS function so the user can tell when the system is getting doses while on the way to steady state.  Without the flag, that `AUC` compartment will keep accumulating stuff and it will end up at an arbitrary amount.  Also, the system will never reach steady state with that AUC compartment given the current criteria (no change in the pre-dose compartment amount ... `dxdt_AUC = CENT/VOLUME;` will always keep increasing).  

So with the flag, I want to freeze that AUC compartment, then advance the system then un-freeze that compartment.  Might also be be some other application.

Reply:
I see. Thanks.